portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,4 +44,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment");
+    testComments.add(comment);
+    response.sendRedirect("/index.html");
```

Comment:
I'm not sure a redirect is what you want here. Usually endpoints like this return an empty response with a 200 status code.

Reply:
The redirect is required (due to the weird implementation in the walkthrough) to move the user back to the page with the comments. Without it the user is stuck on the /data page.

Reply:
Got it, sounds good then